modules/oadp-about-vsl.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-azure.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-gcp.adoc
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+[id="oadp-about-vsl_{context}"]
+= Configuring snapshot locations
+
+The OpenShift API for Data Protection (OADP) backs up persistent volumes (PVs) by using snapshots. Snapshots are stored in snapshot locations, which are specified by the cloud provider parameters.
+
+[IMPORTANT]
+====
+The PVs and the snapshot locations must be in the same region.
```

Comment:
We also have to make a note here that you have to have the default credentials installed for VSL's to work. I don't know where this should go.

Reply:
If this is in the upstream docs, can you provide a link so that I can read it?

Reply:
https://velero.io/docs/v1.7/locations/#limitations--caveats I think this is the list of things that I was trying to convey.

Reply:
Yes, this should be added. Thanks!